### PR TITLE
Add awaiting review to review states in feedback CSV

### PR DESCRIPTION
**Pull request by maureensturgeon**
Update endpoint that powers teacher feedback CSV to include "Waiting for review" as a review state
Screen Shot 2021-08-17 at 7 07 13 PM

## Links

- jira ticket: [LP-2006](https://codedotorg.atlassian.net/browse/LP-2006)

## Testing story
Added a couple of dashboard tests, tested CSV download locally

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/test/controllers/api/v1/assessments_controller_test.rb [hunk 1]
```diff
@@ -689,4 +689,23 @@ class Api::V1::AssessmentsControllerTest < ActionController::TestCase
...
+    lesson_group = create :lesson_group, script: script
+    lesson = create :lesson, script: script, lesson_group: lesson_group
+    weblab_level = create :weblab
+    create :script_level, script: script, levels: [weblab_level], lesson: lesson
+
+    [@student_1, @student_2, @student_3, @student_4].each do |student|
+      create :teacher_feedback, script: script, level: weblab_level, student: student, teacher: @teacher
+    end
+
+    assert_queries 13 do
```
Comment (jamescodeorg) [inline dashboard/test/controllers/api/v1/assessments_controller_test.rb:705]: Mostly curious, why did you add this test?  (I haven't found these assert_queries tests to be very useful?)

File: dashboard/app/models/script.rb [hunk 2]
```diff
@@ -1929,11 +1930,14 @@ def get_feedback_for_section(section)
 
     script_levels.each do |script_level|
       next unless script_level.oldest_active_level.can_have_feedback?
+      current_level = script_level.oldest_active_level
```
Comment (jamescodeorg) [inline dashboard/app/models/script.rb:1932]: (very minor) Maybe swap this with the line above so we don't call `script_level.oldest_active_level` twice?

**Review by jamescodeorg — APPROVED** [review]

File: dashboard/test/controllers/api/v1/assessments_controller_test.rb (hunk 1 above)
Comment (maureensturgeon) [reply dashboard/test/controllers/api/v1/assessments_controller_test.rb:705]: This seems to be our current method of keeping track of how many queries we're producing, which is a way of catching n+1s. I noticed that script_level.oldest_active_level was being called for every student in the section and I was wondering if the result of the query was being cached or if we were re-querying to get the level every time. Turns out we were not re-querying, so that's good.
Summary - I was trying to see if there were unnecessary queries being produced, there weren't. I figured I'd leave it so we can track if changes in the future produce more queries than we intend

File: dashboard/app/models/script.rb (hunk 2 above)
Comment (maureensturgeon) [reply dashboard/app/models/script.rb:1932]: Oh yeah good call
